docs/contributing.rst

```diff
@@ -0,0 +1,9 @@
+.. _contributing:
+
+============
+Contributing
+============
+
+Code of Conduct
+---------------
```

Comment:
I would probably just link to the sunpy one.

Reply:
Okay, sure

Reply:
I ended up doing something a little different as it felt a bit odd to use the same licence for the `Code of Conduct` for `glue-solar` with `sunpy`, so I adapted the `sunpy` one. Hope this arrangement is okay with you.

Reply:
Should be ok.